src/locales/es/pokemon-form.ts

```diff
@@ -0,0 +1,168 @@
+import { SimpleTranslationEntries } from "#app/interfaces/locales";
+
+export const pokemonForm: SimpleTranslationEntries = {
+  // Battle forms
+  "mega": "Mega {{pokemonName}}",
+  "megaX": "Mega {{pokemonName}} X",
+  "megaY": "Mega {{pokemonName}} Y",
+  "primal": "{{pokemonName}} Primigenio",
+  "gigantamax": "G-Max {{pokemonName}}",
+  "eternamax": "E-Max {{pokemonName}}",
+
+  // Starters forms
+  // 1G
+  "pikachuCosplay": "Coqueta",
+  "pikachuCoolCosplay": "Roquera",
+  "pikachuBeautyCosplay": "Aristócrata",
+  "pikachuCuteCosplay": "Superstar",
+  "pikachuSmartCosplay": "Erudita",
+  "pikachuToughCosplay": "Enmascarada",
+  "pikachuPartner": "Compañero",
+  "eeveePartner": "Compañero",
+  // 2G
+  "pichuSpiky": "Spiky",
+  "unownA": "A",
+  "unownB": "B",
+  "unownC": "C",
+  "unownD": "D",
+  "unownE": "E",
+  "unownF": "F",
+  "unownG": "G",
+  "unownH": "H",
+  "unownI": "I",
+  "unownJ": "J",
+  "unownK": "K",
+  "unownL": "L",
+  "unownM": "M",
+  "unownN": "N",
+  "unownO": "O",
+  "unownP": "P",
+  "unownQ": "Q",
+  "unownR": "R",
+  "unownS": "S",
+  "unownT": "T",
+  "unownU": "U",
+  "unownV": "V",
+  "unownW": "W",
+  "unownX": "X",
+  "unownY": "Y",
+  "unownZ": "Z",
+  "unownExclamation": "!",
+  "unownQuestion": "?",
+  // 3G
+  "castformSunny": "Sol",
+  "castformRainy": "Lluvia",
+  "castformSnowy": "Nieve",
+  // 4G
+  "burmyPlant": "Planta",
+  "burmySandy": "Arena",
+  "burmyTrash": "Basura",
+  "shellosEast": "Este",
+  "shellosWest": "Oeste",
+  "rotomHeat": "Calor",
+  "rotomWash": "Lavado",
+  "rotomFrost": "Frío",
+  "rotomFan": "Ventilador",
+  "rotomMow": "Corte",
+  // 5G
+  "basculinRedStriped": "Raya Roja",
+  "basculinBlueStriped": "Raya Azul",
+  "basculinWhiteStriped": "Raya Blanca",
+  "deerlingSpring": "Primavera",
+  "deerlingSummer": "Verano",
+  "deerlingAutumn": "Otoño",
+  "deerlingWinter": "Invierno",
+  // 6G
+  "froakieBattleBond": "Fuerte Afecto",
+  "scatterbugMeadow": "Floral",
+  "scatterbugIcySnow": "Nevado",
+  "scatterbugPolar": "Polar",
+  "scatterbugTundra": "Tundra",
+  "scatterbugContinental": "Continental",
+  "scatterbugGarden": "Vergel",
+  "scatterbugElegant": "Elegante",
+  "scatterbugModern": "Moderno",
+  "scatterbugMarine": "Marino",
+  "scatterbugArchipelago": "Isleño",
+  "scatterbugHighPlains": "Estepa",
+  "scatterbugSandstorm": "Desierto",
+  "scatterbugRiver": "Río",
+  "scatterbugMonsoon": "Borrasca",
+  "scatterbugSavanna": "Sabana",
+  "scatterbugSun": "Solar",
+  "scatterbugOcean": "Océano",
+  "scatterbugJungle": "Jungla",
+  "scatterbugFancy": "Sofisticado",
+  "scatterbugPokeBall": "PokéBall",
```

Comment:
Taking back some of the translations where I did not follow mainline game translations exactly, in order to keep a more universal wording. Now they should be like they are in the mainline "ES" games, poor as I find those translations.

```suggestion
  "scatterbugMeadow": "Floral",
  "scatterbugIcySnow": "Polar",
  "scatterbugPolar": "Taiga",
  "scatterbugTundra": "Tundra",
  "scatterbugContinental": "Continental",
  "scatterbugGarden": "Vergel",
  "scatterbugElegant": "Oriental",
  "scatterbugModern": "Moderno",
  "scatterbugMarine": "Marino",
  "scatterbugArchipelago": "Isleño",
  "scatterbugHighPlains": "Estepa",
  "scatterbugSandstorm": "Desierto",
  "scatterbugRiver": "Oasis",
  "scatterbugMonsoon": "Monzón",
  "scatterbugSavanna": "Pantano",
  "scatterbugSun": "Solar",
  "scatterbugOcean": "Océano",
  "scatterbugJungle": "Jungla",
  "scatterbugFancy": "Fantasía",
  "scatterbugPokeBall": "Poké Ball",
```